controllers/api.controller.js

```diff
@@ -64,3 +64,20 @@ exports.postCommentToArticle = (req, res, next) => {
     })
 };
 
+
+
+exports.patchArticleVotes = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+ 
+  if (!inc_votes || typeof inc_votes !== "number") {
+    return res.status(400).send({ msg: "Invalid or missing inc_votes value" });
+  }
```

Comment:
You don't need to manually check these - let psql deal with it. They will violate the table 'rules' in the seed. Remove this and see what gets passed to your catch block!